include/simdjson/stringparsing.h

```diff
@@ -98,6 +98,16 @@ really_inline  bool parse_string(UNUSED const uint8_t *buf, UNUSED size_t len,
     auto quote_mask = _mm256_cmpeq_epi8(v, _mm256_set1_epi8('"'));
     auto quote_bits =
         static_cast<uint32_t>(_mm256_movemask_epi8(quote_mask));
+#elif defined(__SSE4_2__)
```

Comment:
As mentioned by @geofflangdale, it is unlikely that SSE4.2 is the right target. Maybe SSSE3?

We need PCLMULQDQ, but that's a separate check (__PCLMUL__ I believe).